### DISCUSSION TITLE
%E4%B8%AD%E9%97%B4%E4%BB%B6/redis/%E9%9D%A2%E8%AF%95%E9%A2%98

### DISCUSSION THREAD

**Discussion by giscus** [xiaobaiTech/golang-full-stack#37] [Announcements]
# %E4%B8%AD%E9%97%B4%E4%BB%B6/redis/%E9%9D%A2%E8%AF%95%E9%A2%98

深入探索Golang全栈开发的世界，本网站提供了从基础到高级的全面学习资源，涵盖数据库、微服务、Kubernetes、Docker等关键技术，以及源码分析、开发工具和读书笔记等实用资料，旨在帮助开发者构建扎实的技术基础，掌握现代软件架构的核心原理。

https://golangguide.top/%E4%B8%AD%E9%97%B4%E4%BB%B6/redis/%E9%9D%A2%E8%AF%95%E9%A2%98.html

**Comment by gofurry** [comment]
这一篇里写的 `Redis 的速度比 Memcached 快`，这是不对的。通常情况下Memcached有更高的吞吐和更低的CPU开销。
